tests/functional/database/08-broadcast-upgrade/flow.cylc

```diff
@@ -0,0 +1,5 @@
+[scheduler]
+    allow implicit tasks = True
+[scheduling]
+    [[graph]]
+        R1 = foo
```

Comment:
This test confirms that the workflow can restart, but doesn't confirm that the broadcasted execution retry delays actually work.

Re-using the script example from `database/10` here would confirm the broadcast worked e.g:

```ini
[runtime]
    [[foo]]
        script = """
            if (($CYLC_TASK_TRY_NUMBER < 2 )); then
                false
            fi
        """
        # NOTE: "execution retry delays" set by broadcast
```

Reply:
I've added this in https://github.com/dwsutherland/cylc-flow/pull/11

Reply:
I also modified an existing test that did a similar thing.